composer.json

```diff
@@ -37,15 +37,15 @@
         }
     ],
     "require": {
-        "composer/installers": "^1.9",
+        "composer/installers": "^2",
```

Comment:
Are these updates that are related to the DDEV stuff? 

If not, they should be a separate MR.

Reply:
Done new MR https://github.com/joachim-n/drupal-core-development-project/pull/17 raised

Reply:
Thanks!